Pipfile.lock

```diff
@@ -1283,12 +1286,9 @@
             "version": "==0.5.1"
         },
         "zipp": {
-            "hashes": [
-                "sha256:957cfda87797e389580cb8b9e3870841ca991e2125350677b2ca83a0e99390a3",
-                "sha256:f5812b1e007e48cff63449a5e9f4e7ebea716b4111f9c4f9a645f91d579bf0c4"
-            ],
+            "editable": true,
             "markers": "python_version >= '3.6'",
-            "version": "==3.5.0"
+            "path": "."
```

Comment:
yep, this is borked again. I'll push a fixed Pipfile.lock. We should figure out why this is happening in your environment.

Reply:
I fixed by merging, but if you fix it in your environment, that is probably better. After you've done that, feel free to merge.